spec/tictactoe_spec.rb

```diff
@@ -1,11 +1,46 @@
-require 'spec_helper'
-require 'tictactoe'
+require "spec_helper"
+require "tictactoe"
+require "presenter"
+require "fake_standard_out"
 
 RSpec.describe "TicTacToe" do
+  class FakePresenter
+    def display_board
+      <<~BOARD
+          |  |
+        --+--+--
+          |  |
+        --+--+--
+          |  |
+      BOARD
+    end
+  end
+  class FakeDisplay
+    def initialize(stdout)
+      @stdout = stdout
+    end
+
+    def output(message)
+      @stdout.puts message
+    end
+  end
```

Comment:
This fake display looks a lot like the real display as well. Unless it gives us an advantage over using the real thing (maybe it can?), we should probably remove it.

Reply:
removed fake display